<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QWidget" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="game_start_page">
    <widget class="AcrylicFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="game_main_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>260</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>单词拼写挑战</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="game_rule_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>游戏规则</string>
      </property>
     </widget>
     <widget class="QPushButton" name="game_rank_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>本地排行</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_start">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="game_start_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>100</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>开始游戏</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLineEdit" name="game_name_lineedit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>敢问尊姓大名</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="game_exi_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>420</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <zorder>stackedWidget_start</zorder>
     <zorder>game_main_label</zorder>
     <zorder>game_rule_btn</zorder>
     <zorder>game_rank_btn</zorder>
     <zorder>game_exi_btn</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="game_rule_page">
    <widget class="AcrylicFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="game_rule_ret_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="game_rule_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>250</width>
        <height>320</height>
       </rect>
      </property>
      <property name="text">
       <string>游戏规则:
玩家需要在规定的时间内拼写出最多的单词,拼写出单词后积分会增加，单词越长加分越多。打错单词扣4分。中途退出不记录成绩。时间结束时的积分为最终成绩。排行榜只显示本地积分排行前五名的成绩和昵称。开始游戏时你可以输入你这次游戏的昵称。</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="AcrylicFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="game_edit_frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>270</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="game_line_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>250</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="game_timer_label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>100s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="game_play_ret_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QLabel" name="game_word_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>210</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>word</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="game_score_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>当前积分:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="AcrylicFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="game_rank_ret_btn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="game_rank_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>180</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>排行榜</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="game_rank_scrollArea">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>311</width>
        <height>281</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="game_rank_scrollArea_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>309</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcrylicFrame</class>
   <extends>QFrame</extends>
   <header location="global">acrylicframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
